src/amp-events.js

```diff
@@ -15,17 +15,19 @@
  */
 
 /**
-* Enum used to specify custom Amp Events
-*
-* @enum {string}
-*/
+ * Common AMP events.
+ * @enum {string}
+ */
 export const AmpEvents = {
-  VISIBILITY_CHANGE: 'amp:visibilitychange',
-  DOM_UPDATE: 'amp:dom-update',
-  BUILT: 'amp:built',
-  ATTACHED: 'amp:attached',
-  STUBBED: 'amp:stubbed',
-  LOAD_START: 'amp:load:start',
-  LOAD_END: 'amp:load:end',
-  ERROR: 'amp:error',
+  // Use short, unique strings to reduce bundle size impact.
+  BUILT: 'e1',
```

Comment:
I'm not sure 30b is worth changing these, especially once we strip out the testing ones. Error messages are a special case because they are usually very long messages, these aren't.

Reply:
Fair enough, don't feel strongly enough to fight ya. 😉